src/sentry_plugins/slack/plugin.py

```diff
@@ -69,6 +69,13 @@ def get_config(self, project, **kwargs):
                 'placeholder': 'e.g. #engineering',
                 'required': False,
                 'help': 'Optional #channel name or @user',
+            }, {
+                'name': 'custom_message',
+                'label': 'Custom message',
+                'type': 'string',
+                'placeholder': 'e.g. Hey <!everyone> there is something wrong',
+                'required': False,
+                'help': 'Optional e.g. <!channel> for send notification to everyone in channel',
```

Comment:
Since this legacy Slack integration only sends alerts to one channel, most of our users will have too high of a volume of events to find sending a notification for every exception helpful. While we should definitely keep this functionality in for users like you for whom this is helpful, can we use the following help text instead?

`Optional -  Slack message formatting can be used`

Reply:
Of course.